fluxx.game.php

```diff
@@ -269,8 +273,8 @@ protected function getAllDatas()
   public function getGameProgression()
   {
     // @TODO: compute and return the game progression
-
-    return 0;
+    // with Fluxx, that's always something of a 50/50 chance ?
```

Comment:
Yeah, i'm not sure about progression either. I don't know if it should always be progressing?
In that case, we will need to find something more clever, like (1 - 1/n) with n being the number of turn so far?

Reply:
Seems worth to question Jurica about this?
Don't have any good idea either, just came across this and wanted to mark it as something we need to think of

Reply:
Actually, I think the forum might be a good place for this. I'll check if someone already asked the question or will ask it.

Reply:
There is nothing exactly about this in the forum, but something to keep in mind is that a single player cannot concede until progression reached 50%. 
Anyway, I've asked the question, we'll see: https://forum.boardgamearena.com/viewtopic.php?f=12&t=19271